src/api/views.py

```diff
@@ -8,6 +8,12 @@
 from django.utils import timezone
 import time
```

Comment:
The `time` module is imported but not used in the file. Consider removing this import to clean up the code.